integration-cli/docker_cli_commit_test.go

```diff
@@ -72,6 +72,44 @@ func TestCommitWithoutPause(t *testing.T) {
 	logDone("commit - echo foo and commit the image with --pause=false")
 }
 
+//test commit a paused container should not unpause it after commit
+func TestCommitPausedContainer(t *testing.T) {
+	defer deleteAllContainers()
+	defer unpauseAllContainers()
```

Comment:
Sorry, I think this needs to go before the `deleteAllContainers()`, otherwise the container can't be removed.